config/datatypes_conf.xml.sample

```diff
@@ -696,6 +696,15 @@
     <datatype extension="pttgf" type="galaxy.datatypes.plant_tribes:PlantTribesTargetedGeneFamilies" />
     <datatype extension="pttree" type="galaxy.datatypes.plant_tribes:PlantTribesPhylogeneticTree" />
     -->
+
+    <!-- media datatypes -->
```

Comment:
I think the par datatype is missing here.

Reply:
fixed